### DISCUSSION TITLE
how do i toggle pvp in a claim?

### DISCUSSION THREAD

**Discussion by GamerLoganDean** [GriefPrevention/GriefPrevention#2293] [Q&A]
what do i do to toggle pvp in an area because im trying to make a pvp area in my server. theres no /pvp or /togglepvp and you cant pvp normally in a claim

**Comment by QarthO** [comment]
well thats not a feature of gp. you'll need to use an addon if you want to toggle that in a claim. i believe gpflags has it
https://modrinth.com/plugin/gpflags
Check out their docs for more info, since it is not made by us

**Reply by GamerLoganDean** [reply]
Yeah I figured that. Its kinda stupid its not a feature.

**Reply by QarthO** [reply]
How is that stupid? The feature exists right there in the addon.

**Reply by GamerLoganDean** [reply]
It should exist in the original plugin not as an addon. Like /toggle pvp its simpler.
